src/questions/eligibility.js

```diff
@@ -0,0 +1,136 @@
+import * as React from 'react'
+
+import { FIELD_TYPES, ROUTES, LINKS } from 'consts'
+import { events } from 'analytics'
+import { api } from 'api'
+import { storeFormData } from 'utils'
+import type { Field, Data } from 'types'
+import { ISSUE_QUESTIONS } from './issues'
+
+export const ELIGIBILITY_QUESTIONS: Array<Field> = [
+  {
+    name: 'ELIGIBILITY_INTRO',
+    stage: 1,
+    required: true,
+    type: FIELD_TYPES.DISPLAY,
+    Prompt: <span>Eligibility criteria</span>,
+    Help: (
+      <span>
+        Before we ask you more about what you need help with, we need to check
+        you're eligibile for our service.
+      </span>
+    ),
+    button: { text: 'Continue', Icon: null },
+  },
+  {
+    name: 'CENTRELINK_SUPPORT',
+    stage: 1,
+    effect: async (data: Data) => {
+      if (data.CENTRELINK_SUPPORT) {
+        window.location.href =
+          'https://test-intake.anikalegal.com/intake/form/14/'
+        // Note: Need to change this depending if it's local, test or live.
+        // local site - http://localhost:3001/intake/form/14/
+        // test site - https://test-intake.anikalegal.com/intake/form/14/
+        // live site - https://intake.anikalegal.com/intake/form/14/

```

Comment:
this is not a good idea because it works against the grain of the form framework
what were you trying to achieve here?

Reply:
Implemented the fix in my local commit will push later.